railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt

```diff
@@ -43,3 +44,13 @@
 # of the video).
 # Rails.application.config.active_storage.video_preview_arguments =
 #   "-vf select=eq(n\,0)+eq(key\,1)+gt(scene\,0.015),loop=loop=-1:size=2,trim=start_frame=1 -frames:v 1 -f image2"
+
+# If you're upgrading and haven't set `cookies_serializer` previously, your cookie serializer
+# was `:marshal`. Continue to use that for backward-compatibility with old cookies.
+# If you have configured the serializer elsewhere, you can remove this.
+#
+# To convert all cookies to JSON, use the `:hybrid` formatter.
+# If you're confident all your cookies are JSON formatted, you can switch to the `:json` formatter.
+#
+# See https://guides.rubyonrails.org/action_controller_overview.html#cookies for more information.
+# Rails.application.config.action_dispatch.cookies_serializer = :marshal
```

Comment:
The new default is `:json`, not marshal. So in this case this config should not be in the new_framework_defaults file. If anything we should set this to `:hybrid` to help people upgrading.